<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit_form</class>
 <widget class="QDialog" name="Deposit_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit</string>
  </property>
  <widget class="QPushButton" name="pushButton_get_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get result!</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Replenishment">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>141</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>value</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>304</width>
     <height>217</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Deposit type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Deposit period (Months):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_months">
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_capitalisation">
      <property name="text">
       <string>Capitalization</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Interest rate, %</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_taxrate">
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_interest">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Periodicity of payments 
(every period)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tax rate, %</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Deposit sum</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_deposit_sum">
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_periodicity">
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Common payment:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Interest:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_common_payment">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_interest">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Tax payment:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_common_tax_payment">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_withdraw">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>141</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replenishment list</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdrawals list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_lists">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add lists</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
